constellations/orion/src/tests_integration/class_integration_test.go

```diff
@@ -197,6 +197,7 @@ func createClass(id int) domains.Class {
 			EndDate:         later1,
 			GoogleClassCode: domains.NewNullString("ab12cd34"),
 			FullState:       0,
+			PricePerSession: 50,
```

Comment:
Make this NullUInt and make sure to test null values too

Reply:
Alright, should be done now.